demos/demo-react-ts/src/components/App.tsx

```diff
@@ -175,11 +171,32 @@ function App() {
   cti.broadcastChannel.onmessage = ({
     data,
   }: MessageEvent<{ type: string; payload?: any }>) => {
-    // Send SDK message to HubSpot
-    if (iframeLocation === "window") {
-      const eventHandler = broadcastEventHandlers[data.type];
-      if (eventHandler && cti[eventHandler] && data.payload) {
-        cti[eventHandler](data.payload);
+    // Send SDK message to HubSpot in the calling window
+    if (cti.iframeLocation === "window") {
+      // TODO: Refactor to use eventHandler to invoke the appropriate function
+      // const eventHandler = broadcastEventHandlers[data.type];
+      // cti._cti[eventHandler](data.payload);
```

Comment:
Added TODO to refactor this in the future when we are able to ensure that we're calling the right event handlers using the map.